### DISCUSSION TITLE
Masque Makes Dominos Bars Disappear

### DISCUSSION THREAD

**Discussion by staqz7170** [SFX-WoW/Masque#202] [Issues] [answered]
Hello i just updated both addons Dominos and Masque and when i logged into my account I get messages about failed to load action bars. 

![image](https://user-images.githubusercontent.com/55257243/64815750-83ba1600-d563-11e9-86d1-96bc8ff382af.png)
I hope this helps you in anyway. I have zero background in programming or coding so please excuse my ignorance.

**Comment by StormFX** [comment, marked as answer]
Did you update with the game running?

**Comment by staqz7170** [comment]
tbh i cant remember if i did

**Comment by staqz7170** [comment]
i went ahead and restarted the game and its working now. sorry for wasting your time.
